scripts/boxgame.js

```diff
@@ -0,0 +1,119 @@
+class BoxGameComponent {
+    box = {};
+    score = document.createElement("div");
+    squares = [];
+    container = document.createElement('div');
+    gameAreaEl = document.createElement("div");
+    gamebox = {
+        x: Math.floor(800 / 100),
+        y: Math.floor(200 / 100)
+    };
```

Comment:
why not just use 8 and 2 as uncalculated values and explicitly call them rows / columns rather than x / y (which isnt how they are used)?
this is too tightly coupled with the .gameArea style in the css